### PR TITLE
[fix](statistics)Skip shadow index while analyzing a table.

### PR DESCRIPTION
**Pull request by Jibing-Li**
Skip shadow index while analyzing a table.

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java [hunk 1]
```diff
@@ -961,6 +962,9 @@ public List<Column> getSchemaByIndexId(Long indexId, boolean full) {
     public Set<Column> getSchemaAllIndexes(boolean full) {
         Set<Column> columns = Sets.newHashSet();
         for (Long indexId : indexIdToMeta.keySet()) {
+            if (isShadowIndex(indexId)) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java:965]: use `org.apache.doris.catalog.OlapTable#getVisibleIndex`

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run p0

**Comment by Jibing-Li** [comment]
run feut

**Comment by Jibing-Li** [comment]
run p0

**Comment by Jibing-Li** [comment]
run p0

**Comment by Jibing-Li** [comment]
run p0

**Comment by Jibing-Li** [comment]
run p0

**Review by morrySnow — APPROVED** [review]

**Review by zfr9527 — APPROVED** [review]
LGTM

**Pull request merged**